tailwind.config.js

```diff
@@ -1,6 +1,10 @@
 /** @type {import('tailwindcss').Config} */
 export const content = ["./src/**/*.{js,jsx,ts,tsx}", "./index.html"];
 export const theme = {
-  extend: {},
+  extend: {
+    fontFamily: {
+      sans: ['Open Sans', 'ui-sans-serif', 'system-ui'],
+    },
+  },
```

Comment:
Remove changes from this file, we don't need it.

Reply:
also implemented this request